wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/zh.md

```diff
@@ -0,0 +1,354 @@
+# 参与 osu! 锦标赛的国家
+
+本文列出了参加各种 osu!, osu!taiko, osu!catch and osu!mania锦标赛的国家以及排名前4的国家或地区。
+
+**请注意:** 某些锦标赛没有在此处列出，因为它们允许一个团队内包括两个以上不同国家的玩家参加比赛，或者不代表国家参加比赛。
+
+**表格中会使用的标记**
+
+| 标记   | 含义               |
+|:-----:|--------------------|
+| ![][true] | 参加           |
+| ![][false] | 未参加        |
+
+## 全模式国家排名
+
+| 排名 | 国家                         | 冠军 |  亚军  |  季军  |
+|:----:|-----------------------------|:----:|:------:|:------:|
+| 1    | ![][flag_KR] South Korea    | 7    | 1      | 2      |
```

Comment:
Do you guys normally leave the country name in English?

Reply:
there is no translation of the country names of the web, so i dont translate this. do I need to translate it?

Reply:
I think you should. Does osu-web actually not have translations for country names?

Reply:
osu-web has the country names databased, so they're only in English. I would prefer them to be translated too if it's not too much work

Reply:
ok, ill translate that

Reply:
`Tournaments` looks good, but this article still has the original country names